packages/react-admin-date-picker/package.json

```diff
@@ -7,9 +7,9 @@
         "lib"
     ],
     "dependencies": {
-        "date-fns": "^2.0.0-alpha.37",
         "moment": "^2.24.0",
-        "react-dates": "^21.3.2"
+        "react-dates": "^21.3.2",
+        "react-with-direction": "^1.3.1"
```

Comment:
what is this?

Reply:
An unmet peer dependency of react-dates (see: https://github.com/airbnb/react-dates/blob/master/package.json).